src/locales/it.yaml

```diff
@@ -0,0 +1,360 @@
+      velocity: Velocità
+      api_url: URL API
+      host: Host
+      layout: Layout
+      name: Nome
+      power: Potenza
+      printers: Stampanti
+      requested_speed: Velocità richiesta
+      services: Servizi
+      speed: Velocità
+      flow: Flusso
+      z_offset: Z Offset
+      high: Alto
+      low: Basso
+      'off': Spento
+      used: usato
+      free: libero
+      total_jobs: Totali lavori di stampa
+      longest_job: Lavoro più lungo
+      total_time: Tempo Totale
+      total_time_avg: Tempo Medio
+      total_print_time: Tempo Totale per stampa
+      total_print_time_avg: Tempo Medio per stampa
+      total_filament: '@.capitalize:app.general.table.header.filament_used Totale usato'
+      total_filament_avg: '@.capitalize:app.general.table.header.filament_used Medio usato'
+      variance: Varianza
+      save_as: '@.capitalize:app.general.btn.save Come'
+      edit_preset: '@.capitalize:app.general.btn.edit Preset'
+      add_preset: '@.capitalize:app.general.btn.add Preset'
+      edit_camera: '@.capitalize:app.general.btn.edit Webcam'
+      add_camera: Aggiungi Webcam
+      extrude_length: Lunghezza Estrusione
+      extrude_speed: Velocità Estrusione
+      confirm: Conferma
+    btn:
+      all: Tutti
+      all_off: '@.capitalize:app.general.btn.all @.capitalize:app.general.label.off'
+      quad_gantry_level: QGL
+      add: Aggiungi
+      pause: Pausa
+      reboot: Riavvia
+      shutdown: Spegni
+      remove: Rimuovi
+      remove_all: '@.capitalize:app.general.btn.remove @.capitalize:app.general.btn.all'
+      rename: Rinomina
+      resume: Riprendi
+      cancel: Annulla
+      save: Salva
+      save_as: '@.capitalize:app.general.label.save_as'
+      upload: Carica
+      upload_print: '@.capitalize:app.general.btn.upload e Stampa'
+      preheat: Preriscalda
+      reprint: Ristampa
+      save_restart: '@.capitalize:app.general.btn.save e @.capitalize:app.general.btn.reboot'
+      adjust_layout: >-
+        @.capitalize:app.general.btn.edit layout
+        @.capitalize:app.general.title.home
+      calibrate: Calibra
+      clear_profile: Pulisci Profilo
+      download: Scarica
+      add_printer: '@.capitalize:app.general.btn.add printer'
+      edit: Modifica
+      extrude: Estrudi
+      retract: Ritrai
+      restart_firmware: '@.capitalize:app.general.btn.reboot Firmware'
+      restart_service_klipper: '@.capitalize:app.general.btn.reboot Klipper'
+      restart_service_moonraker: '@.capitalize:app.general.btn.reboot Moonraker'
+      restart_service_webcamd: '@.capitalize:app.general.btn.reboot Webcam'
+      socket_refresh: Ricarica forzata
+      reset_file: Reimposta File
+      reset_layout: Reimposta Layout
+      view: Guarda
+      close: Chiudi
+      config_reference: Riferimento Impostazioni
+      add_file: '@.capitalize:app.general.btn.add File'
+      add_dir: '@.capitalize:app.general.btn.add Directory'
+    table:
+      header:
+        filament_used: Filamento
```

Comment:
Suggestion: I may not have enough context, but I think this should be: `Filamento Utilizzato`

Reply:
Its placed as label for a column in jobs, to me is already clear only with Filamento

Reply:
Yep, I was out of context here